src/app/pages/onboarding/set-password/set-password.tsx

```diff
@@ -154,36 +153,52 @@ export const SetPasswordPage = () => {
                   height="64px"
                   key="password-input"
                   name="password"
+                  onBlur={formik.handleBlur}
                   onChange={formik.handleChange}
                   placeholder="Set a password"
                   type="password"
                   value={formik.values.password}
                 />
-                {formik.submitCount && formik.errors.password ? (
+                {formik.errors.password &&
+                  (formik.errors.password.includes('Weak') ||
+                  formik.errors.password.includes('Strong') ? (
+                    <Stack alignItems="center" isInline>
```

Comment:
The logic here isn't clear to me. Why do we conditionally show this error only if the error contains these words?